Tests/scs-test-runner.py

```diff
@@ -58,17 +59,63 @@ def get_subject_mapping(self, subject):
         mapping.update(self.subjects.get(subject, {}).get('mapping', {}))
         return mapping
 
+    def generate_compliance_check_jobs(self, subjects, scopes):
+        jobs = []
+        for subject in subjects:
+            for scope in scopes:
+                if scope == "scs-compatible-iaas":
+                    jobs.append([scope, subject])
+                if scope == "scs-compatible-kaas":
+                    k8s_setup = self.get_kubernetes_setup(subject)
+                    for k8s_version in k8s_setup["kube_versions"]:
+                        kaas_subject = f"{subject}.{k8s_version}"
+                        jobs.append([scope, kaas_subject, k8s_version, k8s_setup["kube_plugin"]])
+        return jobs
+
+    def get_kubernetes_setup(self, subject):
+        default_kubernetes_setup = self.subjects.get('_', {}).get('kubernetes_setup', {})
+        kubernetes_setup = {"kube_plugin": default_kubernetes_setup['kube_plugin'], "kube_versions": default_kubernetes_setup['kube_versions']}
+        kubernetes_setup.update(self.subjects.get(subject, {}).get('kubernetes_setup', {}))
+        return kubernetes_setup
+
+    def build_clusters_for_jobs_in_sequence(self, jobs):
+        for i in range(len(jobs)):
+            if jobs[i][0] == "scs-compatible-kaas":
+                cluster_id = f"{jobs[i][1]}"
+                logger.debug(f"Provide cluster '{cluster_id}'")
+                kubeconfig_path = os.getcwd() + "/k8s_test_kubeconfigs/"
+                if not os.path.exists(kubeconfig_path):
+                    os.mkdir(kubeconfig_path)
+                kubeconfig_filepath = kubeconfig_path + f"{cluster_id}.yaml"
+                kubeconfig = run_plugin_create(jobs[i][3], cluster_id, jobs[i][2], (kubeconfig_filepath))
+                jobs[i].append(kubeconfig)
+        return jobs
+
+    def delete_clusters_for_jobs_in_sequence(self, jobs):
+        for i in range(len(jobs)):
+            if jobs[i][0] == "scs-compatible-kaas":
+                cluster_id = f"{jobs[i][1]}"
+                logger.debug(f"Delete cluster '{cluster_id}'")
+                run_plugin_delete(jobs[i][3], cluster_id)
```

Comment:
This should go in to `Tests/kaas/plugin/run_plugin.py` as described in my comment for `config.toml`.